src/components/StdoutContext.ts

```diff
@@ -0,0 +1,15 @@
+import {createContext} from 'react';
+
+export interface StdoutProps {
```

Comment:
```suggestion
export interface StdoutContextProps {
```

Reply:
Done in https://github.com/vadimdemedes/ink/pull/264/commits/e75f4d21ccc379e28ab2352359343206314fed9d